src/test/java/ladder/domain/HeightTest.java

```diff
@@ -0,0 +1,34 @@
+package ladder.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class HeightTest {
+
+    @Test
+    @DisplayName("높이는 1 이상의 숫자이다.")
+    void createHeight() {
+        // when & then
+        assertThatCode(() -> new Height("5"))
```

Comment:
이런 경우엔 보통 경계값을 테스트하곤 해요!
```suggestion
        assertThatCode(() -> new Height("1"))
```

Reply:
조앤의 리뷰에 따라 테스트하는 값을 수정하였습니다